cf-runagent/cf-runagent.c

```diff
@@ -368,6 +384,22 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
             LoggingEnableTimestamps(true);
             break;
 
+        /* long options only */
+        case 0:
+
+            if (strcmp(OPTIONS[longopt_idx].name, "remote-bundles") == 0)
+            {
+                size_t len = strlen(REMOTEBUNDLES);
+                StrCatDelim(REMOTEBUNDLES, sizeof(REMOTEBUNDLES), &len,
+                            optarg, ',');
+                if (len >= sizeof(REMOTEBUNDLES))
+                {
+                    Log(LOG_LEVEL_ERR, "Argument too long (-B)");
```

Comment:
(reposting in PR instead of commit)

Also here, is ``-B`` correct if there is only a long option (I might be misunderstanding ``getopt``, I don't remember the exact semantics)?